native/src/routes/search/components/SearchModal.js

```diff
@@ -130,7 +134,7 @@ class SearchModal extends React.Component<PropsType, StateType> {
         <View style={{ minHeight: minHeight }}>
           <CategoryList
             categories={filteredCategories}
-            navigateToLink={navigateToLink}
+            navigateToLink={navigateToLink} // Do i have to add changes to navigateToLink too? When ist this used?
```

Comment:
I think it is not really used anywhere for search but only for links in 2nd level content, so probably no changes required. But please make sure this is really the case by checking usages of `navigateToLInk` in `CategoryList`.